lib/rules/capitalized-comments.js

```diff
@@ -0,0 +1,258 @@
+                                        },
+                                        additionalProperties: false
+                                    },
+                                    block: {
+                                        type: "object",
+                                        properties: {
+                                            capitalize: { enum: ["always", "never"] },
+                                            ignorePattern: {
+                                                type: "string"
+                                            },
+                                            markers: {
+                                                type: "array",
+                                                items: {
+                                                    type: "string"
+                                                },
+                                                uniqueItems: true
+                                            },
+                                            ignoreInlineComments: {
+                                                type: "boolean"
+                                            }
+                                        },
+                                        additionalProperties: false
+                                    }
+                                },
+                                additionalProperties: false
+                            }
+                        },
+                        additionalProperties: false
+                    }
+                ]
+            }
+        ]
+    },
+
+    create(context) {
+
+        const normalizedOptions = getAllNormalizedOptions(context.options[0]),
+            sourceCode = context.getSourceCode();
+
+        //----------------------------------------------------------------------
+        // Helpers
+        //----------------------------------------------------------------------
+
+        /**
+         * Checks whether a comment is an inline comment.
+         *
+         * @param {ASTNode} comment The comment node to check.
+         * @returns {boolean} True if the comment is an inline comment, false
+         * otherwise.
+         */
+        function isInlineComment(comment) {
+            const firstToken = sourceCode.getFirstToken(comment),
+                previousToken = sourceCode.getTokenBefore(firstToken);
+
+            return previousToken &&
+                firstToken.loc.line === previousToken.loc.line;
+        }
+
+        /**
+         * Process a comment to determine if it needs to be reported.
+         *
+         * @param {ASTNode} comment The comment node to process.
+         * @returns {void}
+         */
+        function processComment(comment) {
+            const options = normalizedOptions[comment.type];
+
+            // 1. Check for markers.
+            if (options.markers.some(m => comment.value[0] === m)) {
+                return;
```

Comment:
I think it should check this comment with the content without the matched marker.
`spaced-comment` rule is checking spacing preceded by the matched marker.